src/assets/theme/megaMenu.ts

```diff
@@ -34,5 +34,5 @@ export const megaMenuFooter: MenuTheme = {
   buttonContentHover: COLOR_MAP.NEW_BLUE.PURPLE,
   buttonBorder: COLOR_MAP.GREY_3,
   aboutUsContentAlignment: 'center',
-  aboutUsTextAlignment: 'center',
+  paragraphAlignment: 'center',
```

Comment:
Renaming `aboutUsTextAlignment` to `paragraphAlignment` as this theme is no longer used for just the "About us" part of the mega-menu/footer.